src/actions/rankingActions.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+import * as types from "./actionTypes";
+import api from "../util/api.js"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'api'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)